hw/drivers/sensors/icp101xx/syscfg.yml

```diff
@@ -0,0 +1,52 @@
+# The BSD License (BSD)
+#
+# Copyright (c) 2016 Adafruit Industries
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+syscfg.defs:
+    ICP101XX_SHELL_ITF_TYPE:
+        description: 'Shell interface type for the ICP101XX'
+        value: 1 # I2C
+    ICP101XX_SHELL_ITF_NUM:
+        description: 'Shell interface number for the ICP101XX'
+        value: 0 #I2C0
+    ICP101XX_SHELL_ITF_ADDR:
+        description: 'ICP101XX I2C Address'
+        value: 0x63
+    ICP10114_SHELL_ITF_ADDR:
+        description: 'ICP10114 I2C Address'
+        value: 0x64
+    ICP101XX_ITF_LOCK_TMO:
+        description: 'ICP101XX interface lock timeout in milliseconds'
+        value: 1000 # Max timeout for a full I2C transaction
+    ICP101XX_CLI:
```

Comment:
The CLI should be disabled by default.

Reply:
Thanks, the ICP101XX_CLI is now disabled